website/en/docs/types/functions.md

```diff
@@ -213,6 +213,54 @@ var num: number = method.call(42);
 var str: string = method.call(42);
 ```
 
+### Functions with %checks <a class="toc" id="toc-function-checks" href="#toc-function-checks"></a>
```

Comment:
You can call these functions "Predicate Functions", since you are referring to them later as such. "%checks" can also be in fixed-width font for consistency with the rest of the doc.